ts/Extensions/Annotations/AnnotationDefaults.ts

```diff
@@ -46,6 +46,12 @@ const { defined } = U;
  */
 
 const AnnotationDefaults: AnnotationOptions = {
+    /**
+     * A class name for styling by CSS.
+     *
+     * @type      {string}
+     * @apioption annotations.className
+     */
```

Comment:
I think we could add `@since` param here, as I believe it will be available since next release and it wasn't available previously.